bin/webpack.js

```diff
@@ -29,35 +29,59 @@ try {
 	webpackCliInstalled = false;
 }
 
+let webpackCommandInstalled = false;
+try {
+	require.resolve("webpack-command");
+	webpackCommandInstalled = true;
+} catch (err) {
+	webpackCommandInstalled = false;
+}
+
 if (!webpackCliInstalled) {
```

Comment:
If the user choses to use `webpack-command`, the warning will be prompt if `webpack` has been linked **after** the CLI.